docs/building-images.md

```diff
@@ -120,3 +120,19 @@ as follows:
 Additionally, the following RPMs must be included in the RPMs directory as described above:
 * `elemental-register`
 * `elemental-system-agent`
+
+### SUSE Manager (SUMA)
```

Comment:
Love that you added docs. Only request is to move this up under the Image Definition section above (after Operating Systems), since these are definition items (as compared to the directory structure items).

Reply:
Good call, I didn't even notice the structure. Will fix.